saveutils/savefile/savefile.py

```diff
@@ -52,6 +52,8 @@ def parse_string(self, string: str) -> None:
         """
         self.parse_time = time()
         self.data = json.loads(string)
+        self.raw_size_conversion_factor = len(json.dumps(self.data))/len(string)
```

Comment:
Will do - I'll have to store the raw string size as a property but the rest can be generated at call time, so that will also avoid the performance hit this would have otherwise added.

Reply:
Resolved in 5883de0

Reply:
Left a comment here: https://github.com/MaxWasUnavailable/SaveUtils/commit/5883de050dfdd4ba35328c03cdf794c5ee397177#r116094976

Reply:
variable initialization added in 873dda2